docs/docs/administer/migrate.mdx

```diff
@@ -6,41 +6,56 @@ sidebar_position: 2
 # Migrate from Omnibus to Docker
 
 Chef Infra Client, the configuration system Chef Omnibus relies on, has been
-[scheduled for End-of-Life in 2024](https://docs.chef.io/versions/#supported-commercial-distributions).
-As such, Omnibus-based deployments
-will be deprecated in a future version of Firezone.
+[scheduled for End-of-Life in 2024](https://docs.chef.io/versions/).
+Firezone 0.7 will be the last version to offer Omnibus-based deployments.
+Users are encouraged to migrate to a Docker-based deployment of Firezone using
+this guide.
+
+Existing Omnibus-based deployments of Firezone will continue to function as-is,
+but no officially supported RedHat or Debian packages will be published for
+Firezone 0.8 and above.
+
+See this [GitHub issue tracking discussion
+](https://github.com/firezone/firezone/issues/) for more details.
```

Comment:
@jamilbk this link is broken, doesn't lead to an issue

Reply:
Good catch, fixed